src/api/elastic/elasticService.js

```diff
@@ -4,56 +4,66 @@ const init = require("./init");
 const PropertyService = require("./../services/property");
 const CityService = require("./../services/city");
 const elasticClient = new elasticsearch.Client({
-    hosts: ["http://localhost:9200"],
+    hosts: ["http://localhost:9200"]
 });
 
 module.exports = {
+    restartIndexing(req, res) {
+        this.initService(req, res);
```

Comment:
Shouldn't you wipe data in elastic before initializing?
Everything else looks legit.